### PR TITLE
Add custom twig template dirs

### PR DESCRIPTION
**Pull request by nickvanderzwet**
Add custom twig-template directories for overriding the default twig files.

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | #158 
| License       | MIT
| Doc PR        | -

### CODE REVIEW

**Comment by nickvanderzwet** [comment]
@dunglas Do you haven any comments/improvements?

File: src/Command/GenerateTypesCommand.php [hunk 1]
```diff
@@ -146,7 +146,11 @@ protected function execute(InputInterface $input, OutputInterface $output): void
         $goodRelationsBridge = new GoodRelationsBridge($relations);
         $cardinalitiesExtractor = new CardinalitiesExtractor($graphs, $goodRelationsBridge);
 
-        $loader = new \Twig_Loader_Filesystem(__DIR__.'/../../templates/');
+        $defaultGeneratorTemplatePath = __DIR__.'/../../templates/';
+        $customGeneratorTemplatePaths = $processedConfiguration['generatorTemplates'];
+        $templatePaths = array_merge($customGeneratorTemplatePaths, [$defaultGeneratorTemplatePath]);
```
Comment (dunglas) [inline src/Command/GenerateTypesCommand.php]:
```suggestion
        $customGeneratorTemplatePaths[] = '__DIR__.'/../../templates/';
```

File: src/Command/GenerateTypesCommand.php [hunk 2]
```diff
@@ -146,7 +146,11 @@ protected function execute(InputInterface $input, OutputInterface $output): void
         $goodRelationsBridge = new GoodRelationsBridge($relations);
         $cardinalitiesExtractor = new CardinalitiesExtractor($graphs, $goodRelationsBridge);
 
-        $loader = new \Twig_Loader_Filesystem(__DIR__.'/../../templates/');
+        $defaultGeneratorTemplatePath = __DIR__.'/../../templates/';
```
Comment (dunglas) [inline src/Command/GenerateTypesCommand.php]:
```suggestion
```

**Comment by nickvanderzwet** [comment]
Seems to be some kind of configuration error (Travis)

**Comment by soyuka** [comment]
thansk @nickvanderzwet

**Pull request merged**